plugins/scenario/hu.bme.mit.gamma.scenario.language.ide/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,19 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Gamma Scenario Language IDE
+Bundle-Vendor: BME-FTSRG
+Bundle-Version: 2.3.1.qualifier
```

Comment:
Here and everywhere else:  the Bundle-Version should be `Bundle-Version: 2.5.0.qualifier` for easier handling with other plugins.